learn/safe-core/safe-core-account-abstraction-sdk/relay-kit.md

```diff
@@ -13,47 +15,145 @@ The [Relay Kit](https://github.com/safe-global/account-abstraction-sdk/tree/main
 ### Install dependencies
 
 ```bash
-yarn add @safe-global/relay-kit
+yarn add ethers @safe-global/relay-kit @safe-global/safe-core-sdk @safe-global/safe-core-sdk-types @safe-global/safe-ethers-lib
 ```
 
-### How to use
+### Relay Kit Options
 
 Currently, the Relay Kit is only compatible with the [Gelato relay](https://docs.gelato.network/developer-services/relay). There are 2 ways to use the Gelato relay:
-1.  [Gelato SyncFee](https://docs.gelato.network/developer-services/relay/quick-start/callwithsyncfee)
-2. [Gelato 1Balance](https://docs.gelato.network/developer-services/relay/payment-and-fees/1balance)
+1. [Gelato 1Balance](https://docs.gelato.network/developer-services/relay/payment-and-fees/1balance)
+2. [Gelato SyncFee](https://docs.gelato.network/developer-services/relay/quick-start/callwithsyncfee)
 
-#### Gelato SyncFee
+## Gelato 1Balance
 
-[Gelato SyncFee](https://docs.gelato.network/developer-services/relay/quick-start/callwithsyncfee) is one of the most straightforward ways to use relaying. Gelato SyncFee allows you to execute a transaction and pay the gas fees directly with funds in your Safe, even if you don't have ETH or the native blockchain token.
+[Gelato 1Balance](https://docs.gelato.network/developer-services/relay/payment-and-fees/1balance) allows you to execute transactions using a prepaid deposit. This can be used to sponsor transactions to other Safes or even to use a deposit on Polygon to pay the fees for a wallet on another chain.
 
-```typescript
-import { GelatoRelayAdapter } from '@safe-global/relay-kit'
+For the 1Balance quickstart tutorial, you will use the Gelato relayer to pay for the gas fees on BNB Chain using the Polygon USDC you've deposited into your Gelato 1Balance account.
 
-const relayAdapter = new GelatoRelayAdapter()
+For this tutorial you will need a Safe with a threshold of 1 deployed on BNB Chain. You can create one using [Safe UI](../../quickstart) or [Protocol Kit](./protocol-kit/).
```

Comment:
I would expect that the `Safe UI` link would point to `app.safe.global` and have an additional note with the quickstart link pointing to the tutorial.